src/index.spec.ts

```diff
@@ -1168,12 +1169,45 @@ describe('EppoPrecomputedJSClient E2E test', () => {
 
     expect(td.explain(mockLogger.logAssignment).callCount).toEqual(2);
     expect(td.explain(mockLogger.logAssignment).calls[1]?.args[0]).toMatchObject({
-      subject: 'test-subject',
+      subject: '',
       featureFlag: 'boolean-flag',
       allocation: 'allocation-124',
       variation: 'variation-124',
-      subjectAttributes: { attr1: 'value1' },
+      subjectAttributes: {},
```

Comment:
@sameerank this test above started failing after I bumped the sdk-common dependency. mind taking a look to see if it looks good to you?